BaseLib/FileTools.cpp

```diff
@@ -62,7 +62,7 @@ void truncateFile( std::string const& filename)
  * found.
  */
 static
-std::size_t findLastPathSeparator(std::string const& path)
+std::string::size_type findLastPathSeparator(std::string const& path)
```

Comment:
I think that function should not be used outside of this file since it is rather low-level. Is it used somewhere else?
If not, could you move it to an anonymous namespace?